lib/util.js

```diff
@@ -62,17 +62,26 @@ exports.deprecate = function(fn, msg) {
   }
 
   var warned = false;
+  const warnedLines = new Set();
+
   function deprecated() {
+    var location = (new Error()).stack.match(/.*?\n.*\n.*?\((.*?:\d+):/)[1];
```

Comment:
@silverwind Do you have the change which landed today? This should ideally have an entry for `deprecated`. Are you doing only `new Error().stack`?